deploy/kube-config/influxdb/heapster.yaml

```diff
@@ -24,7 +24,7 @@ spec:
         imagePullPolicy: IfNotPresent
         command:
         - /heapster
-        - --source=kubernetes:https://kubernetes.default
+        - --source=kubernetes.summary_api:''
```

Comment:
please don't do unrelated changes in the same commit.

Reply:
Is the kubelet source still used? If not, I'll add this in another PR.